src/components/Counter/__stories__/counter.stories.scss

```diff
@@ -36,7 +33,7 @@
   &_line {
     width: 40px;
     height: 1px;
-    @include theme-prop(background-color, icon-color-hover);
+    background-color: var(--color-wolf_gray);
```

Comment:
wolf grey? why not using a semantic token herE?

Reply:
That was in theme-mixin, so I'm referencing the same value as it was before
<img width="440" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/b5e4791f-5219-43ea-a8dc-c30bd6215eb0">
<img width="624" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/d3a40095-305a-4611-8a91-367f41f6688d">

Reply:
it's not semantic it's a primitive can you consult with a designer about the fit semantic color?

Reply:
Replaced with Divider component